test/UnbrandedProjects/Customized-TypeSpec/src/Generated/SuperClient.cs

```diff
@@ -414,20 +401,16 @@ public virtual ClientResult CreateLiteral(BinaryContent content, RequestOptions
         }
 
         /// <summary> Send literal parameters. </summary>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<ClientResult<Thing>> HelloLiteralAsync(CancellationToken cancellationToken = default)
+        public virtual async Task<ClientResult<Thing>> HelloLiteralAsync()
         {
-            RequestOptions options = FromCancellationToken(cancellationToken);
-            ClientResult result = await HelloLiteralAsync(options).ConfigureAwait(false);
+            ClientResult result = await HelloLiteralAsync().ConfigureAwait(false);
```

Comment:
this is terribly wrong, because this causes infinite invocation and leads to stack over flow.